### PR TITLE
feat: add flag to not open the browser automatically

### PR DESCRIPTION
**Pull request by alxndr13**
Sometimes i want to authenticate using an incognito window, but calendarsync opens the auth dialog in the last recent window i used. To make sure i can select which browser, window or profile i want to use for the authentication, i can now add the flag `--open-browser false` to get a link which can be copied to the browser or window the user selects.

### CODE REVIEW

File: internal/adapter/outlook_http/adapter.go [hunk 1]
```diff
@@ -161,7 +161,7 @@ func (c *CalendarAPI) SetupOauth2(ctx context.Context, credentials auth.Credenti
 	return nil
 }
 
-func (c *CalendarAPI) Initialize(ctx context.Context, config map[string]interface{}) error {
```
Comment (MichaelEischer) [inline internal/adapter/outlook_http/adapter.go:164]: The variable is never used.

File: internal/adapter/google/adapter.go [hunk 2]
```diff
@@ -106,7 +104,7 @@ func (c *CalendarAPI) SetupOauth2(ctx context.Context, credentials auth.Credenti
 // The given config is presumably unknown and is validated and loaded in order to construct a valid
 // CalendarAPI struct.
 // If anything fails, an error is returned and the CalendarAPI should be considered non-functional.
-func (c *CalendarAPI) Initialize(ctx context.Context, config map[string]interface{}) error {
+func (c *CalendarAPI) Initialize(ctx context.Context, openBrowser bool, config map[string]interface{}) error {
```
Comment (MichaelEischer) [inline internal/adapter/google/adapter.go:107]: The variable is never used.

**Review by MichaelEischer — CHANGES_REQUESTED** [review]
There's something missing here, the `openBrowser` argument is just passed along, but never used.

**Comment by alxndr13** [comment]
@MichaelEischer thanks for the heads up. I had some changes that I hadn't committed yet. They are committed now, please take a look. :) 

I was surprised when you mentioned that this variable wasn't used. I've been using my custom build for four weeks without issues, but I guess I need to commit my changes. 😄

**Review by MichaelEischer — APPROVED** [review]

**Pull request merged**